README-pt-BR.md

```diff
@@ -0,0 +1,347 @@
+
+Nosso objetivo é prover todas as informações que os pesquisadores precisam no início de seus projetos para assegurar que eles serão facilmente reproduzíveis quando terminados.
+
+Isso também significa garantir que os alunos de doutorado, pós-doutorandos, pesquisadores principais e equipes de financiamento saibam quais partes da "responsabilidade de reprodutibilidade" podem afetar e o que devem fazer para estimular a ciência de dados a ser mais eficiente, eficaz e compreensível.
+
+Tabela de conteúdos:
+
+- [Sobre o projeto](#sobre-o-projeto)
+- [O time](#o-time)
+- [Como contribuir](#como-contribuir)
+- [Citando o _The Turing Way_](#citando-o-the-turing-way)
+- [Entre em contato](#entre-em-contato)
+- [Contribuidores](#contribuidores)
+
+🎧 Se você preferir uma introdução audível ao projeto, Rachael, que é membra do nosso time, fez uma apresentação no [Open Science Fair 2019](https://www.opensciencefair.eu/) em Porto e sua demo foi gravada pelo [Orion podcast](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
+A visão geral do _The Turing Way_ começa em decorridos 5:13.
+
+### Sobre o projeto
+
+A pesquisa reproduzível é necessária para garantir que trabalhos científicos sejam confiáveis.
+Financiadores e editores estão começando a exigir que as publicações incluam o acesso aos dados subjacentes e ao código de análise.
+O objetivo é garantir que todos os resultados possam ser verificados de forma independente e aproveitados em trabalhos futuros.
+Às vezes, é mais fácil falar do que fazer.
+Compartilhar esses resultados de pesquisa significa compreender o gerenciamento de dados, o funcionamento das bibliotecas, o desenvolvimento de software e as técnicas de integração contínua: habilidades que não são, de forma ampla, ensinadas ou esperadas de pesquisadores acadêmicos e cientistas de dados.
+Como essas atividades não são comumente ensinadas, reconhecemos que a carga da exigência e a aquisição de novas habilidades podem ser intimidantes para pessoas que são novas neste mundo.
+O *The Turing Way* é um manual para apoiar os alunos, seus supervisores, financiadores e editores de periódicos a garantir que a ciência de dados reproduzível seja "fácil demais para não ser feita", mesmo para pessoas que nunca trabalharam dessa forma antes.
+Ele inclui o material de treinamento sobre controle de versão, teste de análise e comunicação aberta e transparente com usuários futuros, além de estudos de caso e workshops do Instituto Turing(*Turing Institute*).  
+Este projeto é desenvolvido abertamente e todas e quaisquer perguntas, comentários e recomendações são bem-vindas em nosso repositório do GitHub: <https://github.com/alan-turing-institute/the-turing-way>.
+
+### O time
+
+Esta é uma parte do trabalho de planejamento da equipe do projeto no Turing Institute.  
```

Comment:
Estes são alguns membros do projeto realizando planejamentos no Turing Institute.

Reply:
It definitely sounds better! Done!